src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -22,7 +22,9 @@ public struct Conversion : IEquatable<Conversion>
         internal static readonly Conversion ImplicitNullable = new Conversion(ConversionKind.ImplicitNullable);
         internal static readonly Conversion ImplicitReference = new Conversion(ConversionKind.ImplicitReference);
         internal static readonly Conversion ImplicitEnumeration = new Conversion(ConversionKind.ImplicitEnumeration);
+        internal static readonly Conversion ImplicitTupleLiteral = new Conversion(ConversionKind.ImplicitTupleLiteral);
         internal static readonly Conversion ImplicitTuple = new Conversion(ConversionKind.ImplicitTuple);
+        internal static readonly Conversion ExplicitTuple = new Conversion(ConversionKind.ExplicitTuple);
```

Comment:
Can you point to any LDM notes that discuss this `ExplicitTuple` conversion? Alternately, please open an issue "proposing" it.